components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/ConfiguredEndpointModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿// ------------------------------------------------------------
```

Comment:
Do we need this new model? I think PublishedNodesEntryModel can be reused here

Reply:
yes, reused PublishedNodesEntryModel